packages/cli/src/events/WorkflowStatistics.ts

```diff
@@ -114,7 +114,7 @@ export async function workflowExecutionCompleted(
 				workflow_id: workflowId,
 			};
 
-			if (!owner.settings?.firstSuccessfulWorkflowId) {
+			if (!owner.settings?.userActivated) {
```

Comment:
can you explain the change here? 
```suggestion
			if (!owner.settings?.firstSuccessfulWorkflowId && !owner.settings?.userActivated) {
```

Reply:
is `firstSuccessfulWorkflowId` not needed? can we remove it compeltely?

Reply:
`userActivated` is a boolean flag to indicate that a user has successfully ran a workflow in production.

We set it to true once it happens, hence it's placed as part of the workflow statistics hooks, that already has a mechanism to detect this.

Also we save the workflow ID that was first executed.

Reply:
It's still needed to get the workflow name that we show in the modal. I was using the firstWorkflowId in the condition because we did not have the userActivated flag then. At this point we do have the userActivated property set for all users because we ran the migration AddUserActivatedProperty

Reply:
thank you